### ISSUE TITLE
Image-text retrieval result can't reproduce

### ISSUE THREAD

**Issue by liuhl-source** [ChenRocks/UNITER#44]
Thanks for your contribution!
I run your code and can not reproduce the performance in your paper.
There my results in two different setting：
Finetuning Image-text retrival with "train-itm-flickr-base-8gpu.json" 
<img width="417" alt="图片" src="https://user-images.githubusercontent.com/25958403/97099875-db183880-16c8-11eb-912d-70aaada3f41d.png">

Finetuning Image-text retrival with"train-itm-flickr-base-16gpu-hn.json"
<img width="447" alt="图片" src="https://user-images.githubusercontent.com/25958403/97099890-f3885300-16c8-11eb-958d-d9942817396b.png">

There is a gap between the results and those in your paper.  What's the difference between the experiment you do and the code you released? For example,  training steps and learning rate. Thanks!

**Comment by RenShuhuai-Andy** [comment]
I have the same problem. I can reproduce the performance of Image-text retrieval with"train-itm-flickr-base-16gpu-hn.json". 
But for "train-itm-coco-base-16gpu-hn.json", the result is 
![image](https://user-images.githubusercontent.com/30052148/102838806-08b7fe80-443a-11eb-8547-82dbe9d70c8b.png)
, which is worse than the results in your paper.
The above results are produced on 8 V100 gpus.
